state/accountsDB.go

```diff
@@ -1288,12 +1292,16 @@ func (adb *AccountsDB) snapshotUserAccountDataTrie(
 
 		stats.NewSnapshotStarted()
 
+		iteratorChannelsNew := &common.TrieIteratorChannels{
```

Comment:
perhaps we should rename to `iteratorChannelsForDataTries`?

Reply:
done